tests/model_selection/test_hyperband_non_daskarray.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pandas as pd
+
+from dask_ml.model_selection import HyperbandSearchCV
+from dask_ml.datasets import make_classification
+from distributed.utils_test import gen_cluster
+from sklearn.linear_model import SGDClassifier
+
+
+@gen_cluster(client=True)
+def test_pandas(c, s, a, b):
```

Comment:
Could this test go in `test_hyperband.py`?

Reply:
yes let me move it there